tuner/tuner/candidate_gen.py

```diff
@@ -160,6 +212,48 @@ def apply_params(
         )
         return MLIRTransformation(template, modified, embeddable)
 
+    def get_reduction_tile_sizes(self, configuration: Configuration):
+        tile_sizes = [0, 0]
+        tile_sizes.append(configuration.tile_sizes[2])
+        return tile_sizes
+
+    def get_workgroup_tile_sizes(self, configuration: Configuration):
+        tile_sizes = configuration.tile_sizes[:2]
+        tile_sizes.append(0)
```

Comment:
We will probably also have to refactor how we store tile sizes to support other pipelines

Reply:
Yeah, there are a number of additional things that could probably use refactoring, but I left that out of the scope of this PR, since it is already very big. I think it makes sense to refactor this when we move the TD generation to use purely python bindings.